<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="rviz" name="rviz_map" type="rviz" args="-d $(find relative_navigator)/config/rviz/map.rviz" output="screen"/>
  <node pkg="rviz" name="rviz_base" type="rviz" args="-d $(find relative_navigator)/config/rviz/base.rviz" output="screen"/>

  <node name="observe_image_republish" pkg="image_transport" type="republish" args="compressed raw">
      <remap from="in" to="/grasscam/image_raw/" />
      <remap from="out" to="/grasscam/image_raw/" />
  </node>
  <node name="nearest_node_image_republish" pkg="image_transport" type="republish" args="compressed raw">
      <remap from="in" to="/graph_localizer/nearest_node_img/image_raw/" />
      <remap from="out" to="/graph_localizer/nearest_node_img/image_raw/" />
  </node>
  <node name="goal_image_republish" pkg="image_transport" type="republish" args="compressed raw">
      <remap from="in" to="/graph_localizer/goal_img/image_raw/" />
      <remap from="out" to="/graph_localizer/goal_img/image_raw/" />
  </node>
  <node name="first_waypoint_image_republish" pkg="image_transport" type="republish" args="compressed raw">
      <remap from="in" to="/graph_path_planner/first_waypoint_img/image_raw/" />
      <remap from="out" to="/graph_path_planner/first_waypoint_img/image_raw/" />
  </node>

</launch>

